source/core/parameters_lagrangian.cc

```diff
@@ -1056,6 +1056,74 @@ namespace Parameters
       prm.leave_subsection();
     }
 
+    void
+    LagrangianPostProcessing::declare_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection("post-processing");
+      {
+        prm.declare_entry(
+          "Lagrangian post processing",
+          "false",
+          Patterns::Bool(),
+          "State whether Lagrangian post-processing should be performed.");
+
+        prm.declare_entry("calculate particles average velocity",
+                          "false",
+                          Patterns::Bool(),
+                          "Enable calculation of particles average velocity.");
+
+        prm.declare_entry("calculate granular temperature",
+                          "false",
+                          Patterns::Bool(),
+                          "Enable calculation of granular temperature.");
+
+        prm.declare_entry("initial step",
+                          "0.0",
+                          Patterns::Double(),
+                          "Initial step to start Lagrangian post-processing.");
+
+        prm.declare_entry("end step",
+                          "0.0",
+                          Patterns::Double(),
+                          "End step to finish Lagrangian post-processing.");
+
+        prm.declare_entry("output frequency",
+                          "100000",
+                          Patterns::Integer(),
+                          "Post-processing output frequency.");
+
+        prm.declare_entry("particles velocity output name",
+                          "particles_velocity",
+                          Patterns::FileName(),
+                          "File output particles velocity.");
+
+        prm.declare_entry("granular temperature output name",
+                          "granular_temperature",
+                          Patterns::FileName(),
+                          "File output granular temperature.");
+      }
+      prm.leave_subsection();
+    }
+
+    void
+    LagrangianPostProcessing::parse_parameters(ParameterHandler &prm)
+    {
+      prm.enter_subsection("post-processing");
+      {
+        Lagrangian_post_processing = prm.get_bool("Lagrangian post processing");
+        calculate_particles_average_velocity =
+          prm.get_bool("calculate particles average velocity");
+        calculate_granular_temperature =
+          prm.get_bool("calculate granular temperature");
+        initial_step              = prm.get_double("initial step");
+        end_step                  = prm.get_double("end step");
```

Comment:
Why are those doubles? Should they not be unsigned integers since they are iteration numbers?

Reply:
Corrected. :)